### DISCUSSION TITLE
dcmqr

### DISCUSSION THREAD

**Discussion by cbake1** [dcm4che/dcm4che#1098] [Q&A] [answered]
Are their plans to port dcmqr utility from the 2.x platform to 5.x?
Or is it named something else?

**Comment by gunterze** [comment, marked as answer]
there is are separate
- [findscu](https://github.com/dcm4che/dcm4che/blob/master/dcm4che-tool/dcm4che-tool-findscu/README.md)
- [movescu](https://github.com/dcm4che/dcm4che/blob/master/dcm4che-tool/dcm4che-tool-movescu/README.md)
- [getscu](https://github.com/dcm4che/dcm4che/blob/master/dcm4che-tool/dcm4che-tool-getscu/README.md)

utilities.
